docs/application/dotnet/guides/multimedia/media-controller.md

```diff
@@ -41,69 +53,95 @@ To enable your application to use the media controller functionality:
     ```
 
-<a name="get_media"></a>
 ## Updating and Retrieving Information
 
 To update the metadata and playback information on the server side, and to retrieve it on the client side:
 
--   To update the metadata and playback information on the server side:
+-   To update the metadata and playback information on the server side, follow these steps:
     1.  Start the media control server using the `Start()` method of the [Tizen.Multimedia.Remoting.MediaControlServer](https://samsung.github.io/TizenFX/latest/api/Tizen.Multimedia.Remoting.MediaControlServer.html) class:
 
-        ```
+        ```csharp
         MediaControlServer.Start();
         ```
 
     2. Update the metadata or playback information using the `SetMetadata()` or `SetPlaybackState()` methods:
 
-        ```
+        ```csharp
         MediaControlServer.SetPlaybackState(MediaControlPlaybackState.Playing, currentPosition);
         ```
 
     3. When no longer needed, stop the media control server using the `Stop()` method:
 
-        ```
+        ```csharp
         MediaControlServer.Stop();
         ```
 
-- To retrieve the metadata and playback information on the client side:
+- To retrieve the metadata and playback information on the client side, follow these steps:
     1.  Create a new instance of the [Tizen.Multimedia.Remoting.MediaControllerManager](https://samsung.github.io/TizenFX/latest/api/Tizen.Multimedia.Remoting.MediaControllerManager.html) class:
 
-        ```
+        ```csharp
         var mediaControllerManager = new MediaControllerManager();
         ```
 
     2. Retrieve the currently-active controllers and select one:
 
-        ```
+        ```csharp
         var controller = mediaControllerManager.GetActiveControllers().First();
         ```
 
     3. Retrieve the metadata or playback information from the server using the `GetMetadata()`, `GetPlaybackState()`, or `GetPlaybackPosition()` methods.
 
         For example, to retrieve the playback state from the server:
 
-        ```
+        ```csharp
         Tizen.Log.info(LogTag, $"Playback state is {controller.GetPlaybackState()}");
         ```
 
-<a name="send_media"></a>
+## Updating and Retrieving Playlist
+
+To create a playlist from the server side, create a new instance of the [Tizen.Multimedia.Remoting.MediaControlPlaylist](https://samsung.github.io/TizenFX/latest/api/Tizen.Multimedia.Remoting.MediaControlPlaylist.html) class:
+
+```csharp
+var playlist = new MediaControlPlaylist("MyFavorite");
+```
+
+You can also create playlist with Metadata using the following code:
+
+```csharp
+var playlist = new MediaControlPlaylist("MyFavoriate",xx
+	new Dictionary<string, MediaControlMetadata>()
+	{
+		{"Song1", new MediaControlMetadata() {Author = "Someone"} },
+		{"Song2", new MediaControlMetadata()}
+	}
+);
+```
+
+To retrieve the playlist and metadata information on the client side, use the following code:
+```csharp
+	controller.PlaylistUpdated += (s, e) =>
```

Comment:
Please use 4 spaces, instead of a tab.

And I think we can remove first 4 spaces of this code example.

---

controller.PlaylistUpdated += (s, e) =>
{
_(4 spaces)_ Tizen.Log.Info(LogTag, $"Updated mode : {e.Mode}, Name : {e.Name}, Title : {e.Playlist.GetMetadata("IDX1").Title}");
};

Reply:
Done